### PR TITLE
Test cases for HHH-9343

### PR DESCRIPTION
**Pull request by codylerum**

### CODE REVIEW

File: hibernate-entitymanager/src/test/java/org/hibernate/jpa/test/criteria/simplecase/BasicSimpleCaseTest.java [hunk 1]
```diff
@@ -78,6 +82,52 @@ public void testCaseInOrderBy2() {
 
 	}
 
+	@Test
+	@TestForIssue(jiraKey = "HHH-9343")
+	public void testCaseStringResult() {
+		EntityManager em = getOrCreateEntityManager();
+		em.getTransaction().begin();
```
Comment (andrei-ivanov) [inline hibernate-entitymanager/src/test/java/org/hibernate/jpa/test/criteria/simplecase/BasicSimpleCaseTest.java]: should there be a commit at the end?

**Comment by codylerum** [comment]
@andrei-ivanov This is/was my first test case so I just copied the others from the class as an example and neither of them do. I'll be happy to add it, but if so should be it added to the others?

File: hibernate-entitymanager/src/test/java/org/hibernate/jpa/test/criteria/simplecase/BasicSimpleCaseTest.java (hunk 1 above)
Comment (andrei-ivanov) [inline hibernate-entitymanager/src/test/java/org/hibernate/jpa/test/criteria/simplecase/BasicSimpleCaseTest.java]: Hm, no clue :-)
Just noticed that it was missing...

**Comment by dreab8** [comment]
thanks @codylerum,
i have integrate the 2 tests and solved the issue

**Pull request closed without merging**